### PR TITLE
Automatic ETL from Albion Bin Data

### PR DESCRIPTION
**Pull request by mawburn**
So, what this does is this runs every 2hrs. First it checks for the latest commit hash on the world.json file. If it's different, then it starts updating the data.

The zones are really the only ones we care about integrity on. Everything else gets truncated before it gets messed with because it's too hard to keep those up to date and don't really need to be updated anyway. 

Other than that, it's pretty straightforward... albeit a little messy. 

With this, we have meta data on all zones. From here we can just give users meta data about a zone when they click it and/or start building out a map they can pull up.

I couldn't find the levels of mobs on chests. I guess that will just have to be part of intel when I add that or maybe I can find somewhere to crowd source it.

### CODE REVIEW

File: packages/api-server/db_migrations/006_zone_info.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
+ALTER TABLE zones ADD is_deep_road BOOLEAN NOT NULL DEFAULT FALSE;
```
Comment (mmcken3) [inline packages/api-server/db_migrations/006_zone_info.sql:1]: I should have asked this on earlier commits, but does the migration tool you are using create transactions for you, or is there a reason for not putting each new migration file in a transaction? Just a safeguard that will likely never get hit but is nice to have

**Comment by mmcken3** [comment]
if the zones get loaded into redis, are they every accessed from the database after the get loaded into the redis instance? Its not a big loss to store it in both places, but if they are only used from the db to load into redis, there might be no need for keeping them in db? Unless something I missing from before with portal connections touches that part of the db

File: packages/api-server/db_migrations/006_zone_info.sql (hunk 1 above)
Comment (mawburn) [reply packages/api-server/db_migrations/006_zone_info.sql:1]: I'm not sure what you mean. You mean by terminating the line with a `;`? 

AFAIK, a transaction is an open connection to the db. This should be creating a transaction in that sense, but I could be misunderstanding.

**Comment by mawburn** [comment]
> if the zones get loaded into redis, are they every accessed from the database after the get loaded into the redis instance?

Persistence and key generation. I have my own `serial` keys that are used to reference the tables together. 

I also added a server_logs table so I can log the hash of the last git hash processed so I don't have to do it every time the server restarts.

**Pull request merged**